src/xenia/cpu/ppc/ppc_emit_memory.cc

```diff
@@ -732,8 +735,9 @@ int InstrEmit_stwcx(PPCHIRBuilder& f, const InstrData& i) {
 
   Value* ea = CalculateEA_0(f, i.X.RA, i.X.RB);
   Value* rt = f.ByteSwap(f.Truncate(f.LoadGPR(i.X.RT), INT32_TYPE));
-  f.Store(ea, rt);
-  f.StoreContext(offsetof(PPCContext, cr0.cr0_eq), f.LoadConstantInt8(1));
+  Value* res = f.ByteSwap(f.Truncate(f.LoadReserved(), INT32_TYPE));
+  Value* v = f.AtomicCompareExchange(ea, res, rt);
```

Comment:
Yeah - they're working as intended - though these suffer from the ABA problem it doesn't matter cause they're still under the global lock.

I've found that, even with the ABA problem, games do not appear to become any more unstable than they already are. Probably a side effect of being developed on PC.

I intended to allow users to disable the global lock in these blocks because that seems to cause a few hang-ups.